examples/tensorflow/image-classifier-resnet50/requirements.txt

```diff
@@ -0,0 +1 @@
+imageio==2.9.0
```

Comment:
nit: It's probably safe to use `imageio==2.9.*`, so we can get any bugfixes that they might release, without risking them changing the API from under us

Reply:
Yep, I agree.